smartcar/smartcar.py

```diff
@@ -65,16 +65,16 @@ def __init__(self, client_id, client_secret, redirect_uri, scope=None, developme
                 or declines the application's permissions. This URL must also be
                 present in the Redirect URIs field in the application dashboard
             scope (bool, optional): A list of permissions requested by the application
-            development (bool, optional): If True, deplays the Mock OEM for testing.
-                Defaults to False
+            test_mode (bool, optional): Launch smartcar auth flow in test mode. Defaults to false.
```

Comment:
nit: capitalization: "smartcar" --> "Smartcar"

Reply:
nit: "Launch **the** Smartcar auth flow in test mode." to be consistent with Node SDK.